torch/nn/functional.py

```diff
@@ -4313,9 +4314,8 @@ def pairwise_distance(x1: Tensor, x2: Tensor, p: float = 2.0, eps: float = 1e-6,
 
 Args:
     x1 (Tensor): First input.
-    x2 (Tensor): Second input (with the same number of dimensions as x1, matching x1 size at dimension `dim`,
-        and broadcastable with x1 at other dimensions).
-    dim (int, optional): Dimension of vectors. Default: 1
+    x2 (Tensor): Second input.
+    dim (int, optional): Dimension along which cosine similarity is computed. Default: 1
```

Comment:
aren't the docs wrong below?

- `Input` isn't a thing, there are two inputs.
- `D` is now based on the output shape, not the "Input"
- The notation doesn't make much sense, are `\ast_1\` and \`ast_2\` not supposed to be the same thing?  They can't be if D is a value...